k6/vu.go

```diff
@@ -1,4 +1,4 @@
-package tests
+package k6
```

Comment:
Shouldn't test specific code live in a test package so that it doesn't get compiled into the binary?

Reply:
maybe a package called `k6_test` or `k6test`?

Reply:
k6 pkg is not for testing, but I'll see what I can do.